src/lib.rs

```diff
@@ -54,6 +54,9 @@ pub mod redis;
 #[cfg(feature = "trufflesuite_ganachecli")]
 #[cfg_attr(docsrs, doc(cfg(feature = "trufflesuite_ganachecli")))]
 pub mod trufflesuite_ganachecli;
+#[cfg(feature = "victoriametrics")]
+#[cfg_attr(docsrs, doc(cfg(feature = "victoriametrics")))]
```

Comment:
What do you think of `victoria-metrics` as feature name?

Reply:
Sure, thank you. Do you know if I only need to change the two lines?

```rust
#[cfg(feature = "victoria-metrics")]
#[cfg_attr(docsrs, doc(cfg(feature = "victoria-metrics")))]
pub mod victoriametrics;
```

Reply:
@DDtKey I've updated the feature to use snake case everywhere.

Reply:
Thank you!